data/PA/incentives.json

```diff
@@ -0,0 +1,1023 @@
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "PA-37",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 450,
+      "maximum": 450
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $450 rebate for air source heat pump. Rebate value depends on equipment efficiency."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "PA-38",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 400
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$400 rebate per outdoor unit for ductless mini-split heat pump with SEER2 of 15.2, EER2 of 11.7, and HSPF2 of 7.8."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "PA-39",
+    "authority_type": "utility",
+    "authority": "pa-peco",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_clothes_dryer",
+    "program": "pa_pECO_applianceRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 15
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$15 rebate for ENERGY STAR certified electric clothes dryers."
+    },
+    "start_date": "2024",
+    "end_date": "2024"
+  },
+  {
+    "id": "PA-40",
+    "authority_type": "utility",
+    "authority": "pa-peco",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_clothes_dryer",
+    "program": "pa_pECO_applianceRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 75
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$75 rebate for heat pump clothes dryers."
+    },
+    "start_date": "2024",
+    "end_date": "2024"
+  },
+  {
+    "id": "PA-41",
+    "authority_type": "utility",
+    "authority": "pa-peco",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_water_heater",
+    "program": "pa_pECO_waterHeatingRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 350
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$350 rebate for ENERGY STAR certified heat pump hot water heaters."
+    },
+    "start_date": "2024",
+    "end_date": "2024"
+  },
+  {
+    "id": "PA-42",
+    "authority_type": "utility",
+    "authority": "pa-peco",
+    "payment_methods": [
+      "pos_rebate"
```

Comment:
Is this really an upfront discount? If not it should be `rebate`. (Same question for the few other `pos_rebate` incentives.)